worker-default-configs/config/cfg~caf~worker~RabbitConfiguration.js

```diff
@@ -21,5 +21,5 @@
     rabbitHost: getenv("CAF_RABBITMQ_HOST") || "rabbitmq",
     rabbitPort: getenv("CAF_RABBITMQ_PORT") || 5672,
     rabbitUser: getenv("CAF_RABBITMQ_USERNAME") || "guest",
-    rabbitPassword: getenv("CAF_RABBITMQ_PASSWORD") || "guest"
+    rabbitPassword: getenvorprop("CAF_RABBITMQ_PASSWORD", "CAF.") || "guest"
```

Comment:
Could we just use getprop, since all envs will be convert to properties?

Reply:
I think that would be a breaking change, as currently we support just providing the `CAF_RABBITMQ_PASSWORD` env var (i.e. just as it is, not as a converted file-based secret) - lots of the integration tests do this.

If were to change this to getprop only, then it forces all workers to use this new property-based mechanism, and would break any worker that just passed `CAF_RABBITMQ_PASSWORD=something`.

Using `getenvorprop` avoids the breaking change.

Reply:
Maybe a clearer way to implement this might be:

```js
rabbitPassword: getenv("CAF_RABBITMQ_PASSWORD") || getprop("CAF.CAF_RABBITMQ_PASSWORD") || "guest"
```

Reply:
If you create a new function like `getSecret()` then you can encapsulate the logic inside it:

```suggestion
    rabbitPassword: getSecret("CAF_RABBITMQ_PASSWORD") || "guest"
```

So that function could then decide internally whether to get the secret directly from the environment variable, or fallback to appending `_FILE` and reading it from disk, or fallback to prepending `CAF.` and reading it from system property, or in future perhaps fallback to reading it directly from Vault, ...

Reply:
Yeah that sounds good to me, I'll update the PRs and see what that looks like

Reply:
Looks to work well thanks, I'll assign out the initial MRs for proper review.

Reply:
Sorry, I didn't see your ticket on the comment regarding these approaches, I'll reply on the ticket now